ts/parts/Tooltip.ts

```diff
@@ -1689,6 +1729,52 @@ class Tooltip {
         }
     }
 
+    /**
+     * If the `stickOnContact` option is active, this will add a tracker shape.
+     *
+     * @private
+     * @function Highcharts.Tooltip#stickOnContact
+     */
+    private stickOnContact(): void {
+        const tooltip = this;
+
+        if (tooltip.options.followPointer || !tooltip.options.stickOnContact) {
+            if (tooltip.stickOnContactTracker) {
+                tooltip.stickOnContactTracker.destroy();
+            }
+            return;
+        }
+
+        const chart = tooltip.chart;
+        const label = tooltip.label;
+        const point = chart.hoverPoint;
+
+        if (!label || !point) {
+            return;
+        }
+
+        // When the mouse pointer is between the anchor point and the label,
+        // the label should stick.
+        const anchorPos = this.getAnchor(point);
+        const labelBBox = label.getBBox();
+
+        // Combine anchor and tooltip
+        const size: Highcharts.SizeObject = {
+            width: Math.max((anchorPos[0] + chart.plotLeft - label.translateX), labelBBox.width),
+            height: Math.max((anchorPos[1] + chart.plotTop - label.translateY), labelBBox.height)
+        };
+
+        if (!tooltip.stickOnContactTracker) {
+            tooltip.stickOnContactTracker = label.renderer
+                .rect(0, 0, size.height, size.width)
+                .attr({ fill: 'rgba(0,0,0,0)' })
```

Comment:
We need to consider styled mode:
1. Add a class name, presumably `.highcharts-tracker`, and add the fill to highcharts.scss.
2. Make the `.attr` conditional, checking if `chart.styledMode`.
See similar code in the Series `drawTracker` function.